sdk/src/trace/tracer_provider.cc

```diff
@@ -23,6 +23,9 @@ nostd::shared_ptr<opentelemetry::trace::Tracer> TracerProvider::GetTracer(
     nostd::string_view library_name,
     nostd::string_view library_version) noexcept
 {
+  if (library_name.data() == nullptr) {
```

Comment:
Passing a nullptr to GetTrace() is the application code issue, and the right behavior should be to let it crash so that the application developer can fix this. Probably I need to understand the spec change more clearly.

Reply:
The latest [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md) mentioned this in explicit that null name should not crash and should be treat like empty name.
> In case an invalid name (null or empty string) is specified, a working Tracer implementation MUST be returned as a fallback rather than returning null or throwing an exception, its name property SHOULD be set to an empty string,